docs/collab/mult_auth.md

```diff
@@ -1,30 +1,30 @@
 # Multiple authors in the mbed Online Compiler
 
-Repositories on developer.mbed.org are not limited to one author for each repository. Instead, several authors may have the right to publish to the same repository, allowing them to collaborate on a project while other users, who may be interested in using the project's code but are not concerned with the individual authors, can interact with a single repository.
+Repositories on [developer.mbed.org](developer.mbed.org) are not limited to one author for each repository. Instead, several authors may have the right to publish to the same repository, allowing them to collaborate on a project. Other users, who may be interested in using the project's code but are not concerned with the individual authors, can interact with a single repository.
```

Comment:
Links always need http:// or https://, otherwise they're added to the end of the current URL and you get a 404